CHANGELOG.md

```diff
@@ -17,6 +17,7 @@ NOTE: [`epaint`](epaint/CHANGELOG.md), [`eframe`](eframe/CHANGELOG.md), [`egui_w
 * `ClippedMesh` has been replaced with `ClippedPrimitive` ([#1351](https://github.com/emilk/egui/pull/1351)).
 * Renamed `Frame::margin` to `Frame::inner_margin`.
 * Renamed `AlphaImage` to `FontImage` to discourage any other use for it ([#1412](https://github.com/emilk/egui/pull/1412)).
+* `dark-light` is not an opt-in feature for `eframe` and `egui_glow`.
```

Comment:
```suggestion
* `dark-light` (dark mode detection) is now an opt-in feature for `eframe` and `egui_glow` ([#1437](https://github.com/emilk/egui/pull/1437)).
```

Reply:
For some reason I can't click the "Commit suggestion" button - I updated the CHANGELOG.md manually so that it reflects your suggested change. Apologies if I am doing this wrong or missing something obvious - I am a relative neophyte to PRs.

Reply:
I couldn't click "Commit suggestion" either, with no explanation from GitHub. May be another GitHub bug 🤷